cmd/scheduler/app/options/options.go

```diff
@@ -68,11 +73,21 @@ type ServerOption struct {
 	PercentageOfNodesToFind    int32
 
 	NodeSelector []string
+	// EventHandler holds configuration for event handler related features.
+	EventHandler EventHandlerConfiguration
 }
 
 // ServerOpts server options.
 var ServerOpts *ServerOption
 
+// EventHandlerConfiguration contains basically elements about event handler.
+type EventHandlerConfiguration struct {
+	// Name is the handler for event processing.
+	Name string
+	// ConfigFile is the path to the event handler configuration file.
+	ConfigFile string
```

Comment:
what's the content of the ConfigFile?

Reply:
developer custom defined according to external eventhandler requirements